.github/dplasma/action.yml

```diff
@@ -0,0 +1,59 @@
+name: DPLASMA
+description: Uses the DPLASMA CI as a tester for PaRSEC
+
+# Environment variables required:
+#      BUILD_TYPE:  ${{matrix.build_type}}
+#      SHARED_TYPE: ${{matrix.shared_type}}
+#      DEVICE: ${{matrix.device}}
+#      PARSEC_INSTALL_DIR: 
+#      INSTALL_DIRECTORY: ${{github.workspace}}/dplasma/install
+#      SETUP: ${{github.workspace}}/dplasma/.github/workflows/env_setup.sh
+
+runs:
+  using: "composite"
+  steps:
+    - name: Setup Build Environment
+      shell: bash
+      run: |
+          echo "SETUP=$PWD/dplasma/.github/workflows/env_setup.sh" >> $GITHUB_ENV
+          echo "INSTALL_DIRECTORY=$PWD/dplasma/install" >> $GITHUB_ENV
+          git clone https://github.com/ICLDisco/dplasma
+          cd dplasma
+          rm -rf parsec
```

Comment:
you can safely remove this line

Reply:
I think you mean the "rm -rf parsec".  I mainly wanted that for testing to ensure that I was really using the previously-built parsec.  It is removed now.